### PR TITLE
feat: use test-suite to run platform tests

### PR DESCRIPTION
**Pull request by shuplenkov**
## Issue being fixed or feature implemented

Use test-suite to run platform tests

## What was done?

Added some helper scripts, modified travis yaml.

## How Has This Been Tested?

With travis

## Breaking Changes

None

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added or updated relevant unit/integration/functional/e2e tests
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,7 +30,7 @@
   },
   "homepage": "https://github.com/dashevo/DashJS#readme",
   "dependencies": {
-    "@dashevo/dapi-client": "~0.14.0-dev.7",
+    "@dashevo/dapi-client": "~0.14.0-dev.8",
```
Comment (shumkov) [inline package.json:33]: I would do that in a separate PR coz you need to update Client options as well.

File: src/SDK/Client/Client.ts [hunk 2]
```diff
@@ -95,6 +95,7 @@ export class Client {
```
Comment (shumkov) [inline src/SDK/Client/Client.ts:95]: Please rename `addresses` option to `dapiAddresses`

Comment (shuplenkov) [reply src/SDK/Client/Client.ts:95]: Why?

Comment (shumkov) [reply src/SDK/Client/Client.ts:95]: I see #110

File: src/SDK/Client/Client.ts [hunk 3]
```diff
@@ -95,6 +95,7 @@ export class Client {
 
             this.wallet = new Wallet({
```
Comment (shumkov) [inline src/SDK/Client/Client.ts:96]: We need to check that if `this.options.wallet.network` is defined it should be equal to `this.network`.
